detox/test/src/Screens/CanvasScreenshotScreen.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import { View, Dimensions, TouchableHighlight } from 'react-native';
+import { Canvas, Rect } from '@shopify/react-native-skia';
+
+const screenWidth = Dimensions.get('window').width;
+const screenHeight = Dimensions.get('window').height;
+
+class ArtisticRectangle extends Component {
+  static defaultProps = {
+    borderSizeV: 12,
+    borderSizeH: 12
+  };
+
+  // there are two skia canvas on purpose to see if the screenshot mechanism 
+  // can handle multiple on different levels
+  render() {
+    const paddingHorizontal = this.props.borderSizeH;
+    const paddingVertical = this.props.borderSizeV;
+    return (
+      <View testID={this.props.testID} accessible={true}>
+        <View style={{ paddingHorizontal, paddingVertical, backgroundColor: 'cyan' }}>
+          <View style={{ paddingHorizontal, paddingVertical, backgroundColor: 'magenta' }}>
+            <View style={{ paddingHorizontal, paddingVertical, backgroundColor: 'yellow' }}>
+              <Canvas style={{ paddingHorizontal, paddingVertical }}>
+                <Rect x={0} y={0} width={10} height={10} color={'black'} />
+              </Canvas>
+            </View>
+            <Canvas style={{ paddingHorizontal, paddingVertical }}>
+              <Rect x={10} y={10} width={10} height={10} color={'black'} />
+            </Canvas>
+          </View>
+        </View>
```

Comment:
This is mostly a copy of "ElementScreenshotScreen"

Just the black bar is re with 2 skia painted squares:

I intentionally do not draw everything in skia, to check if painting both together works well